go-delegator/function_/queries.go

```diff
@@ -0,0 +1,15 @@
+package function_
```

Comment:
Is `function` a go reserved keyword? According to https://go.dev/ref/spec#Keywords it is not 🤔

Reply:
Oh I thought it was reserved, just like error in api
thank you for pointing out